src/apportioned-emissions/hourly/hourly-apportioned-emissions.service.ts

```diff
@@ -66,169 +68,102 @@ export class HourlyApportionedEmissionsService {
-  //         type: req.headers.accept,
-  //         disposition: `attachment; filename="hourly-emissions-state-aggregation-${uuid()}.csv"`,
-  //       });
-  //     }
-
-  //     const objToString = new PlainToJSON();
-  //     return new StreamableFile(stream.pipe(toDto).pipe(objToString), {
-  //       type: req.headers.accept,
-  //       disposition: `attachment; filename="hourly-emissions-state-aggregation-${uuid()}.json"`,
-  //     });
-  //   } catch (e) {
-  //     console.log(e);
-  //     return null;
-  //   }
-  // }
-
-  // async streamEmissionsNationalAggregation(
-  //   req: Request,
-  //   params: HourlyApportionedEmissionsParamsDTO,
-  // ): Promise<StreamableFile> {
-  //   try {
-  //     const query = this.repository.getNationalStreamQuery(params);
-  //     let stream: ReadStream = await this.streamService.getStream(query);
-
-  //     req.on('close', () => {
-  //       stream.emit('end');
-  //     });
-
-  //     req.res.setHeader(
-  //       fieldMappingHeader,
-  //       JSON.stringify(
-  //         fieldMappings.emissions.hourly.data.aggregation.national,
-  //       ),
-  //     );
-
-  //     const toDto = new Transform({
-  //       objectMode: true,
-  //       transform(data, _enc, callback) {
-  //         const dto = plainToClass(
-  //           HourlyApportionedEmissionsNationalAggregationDTO,
-  //           data,
-  //           {
-  //             enableImplicitConversion: true,
-  //           },
-  //         );
-  //         const date = new Date(dto.date);
-  //         dto.date = date.toISOString().split('T')[0];
-  //         callback(null, dto);
-  //       },
-  //     });
-
-  //     if (req.headers.accept === 'text/csv') {
-  //       const toCSV = new PlainToCSV(
-  //         fieldMappings.emissions.hourly.data.aggregation.national,
-  //       );
-  //       return new StreamableFile(stream.pipe(toDto).pipe(toCSV), {
-  //         type: req.headers.accept,
-  //         disposition: `attachment; filename="hourly-emissions-national-aggregation-${uuid()}.csv"`,
-  //       });
-  //     }
-
-  //     const objToString = new PlainToJSON();
-  //     return new StreamableFile(stream.pipe(toDto).pipe(objToString), {
-  //       type: req.headers.accept,
-  //       disposition: `attachment; filename="hourly-emissions-national-aggregation-${uuid()}.json"`,
-  //     });
-  //   } catch (e) {
-  //     console.log(e);
-  //     return null;
-  //   }
-  // }
+  streamEmissionsFacilityAggregation(
+    req: Request,
+    params: StreamHourlyApportionedEmissionsParamsDTO,
+  ): Promise<StreamableFile> {
+    const disposition = `attachment; filename="hourly-emissions-facility-aggregation-${uuid()}"`
+    const [sql, values] = this.repository.getFacilityStreamQuery(params);
+    const toDto = new Transform({
+      objectMode: true,
+      transform(data, _enc, callback) {
+        data = exclude(data, params, ExcludeApportionedEmissions);
```

Comment:
Don't need to exclude since the parameter is not present for aggregation endpoints. Same for all aggregation